packages/keystone/logging/getKeystonePinoOptions.js

```diff
@@ -14,6 +15,12 @@ function getKeystonePinoOptions () {
         customProps: (req, res) => {
             return getReqLoggerContext(req)
         },
+        serializers: {
+            res: (res) => {
+                const errors = res.raw.req.errors
+                return { gqlErrors: errors, ...serializers.req(res) }
```

Comment:
Hm, is it correct behaviour? (You're overriding` serializers.res`, but using `.req` here)